packages/shared/src/styles/base.css

```
@@ -14,6 +14,8 @@ body {
   min-height: 100vh;
   /* mobile viewport bug fix */
   min-height: stretch;
+  /* disable scroll bounce */
+  overscroll-behaviour: none;
```

The property is in US english that's why it says the property is not supported since the spelling wasn't there 😅 
```suggestion
  overscroll-behavior: none;
```

---

i feel so stupid right now 😂

---

I've felt that countless times in my life 😂